lib/happo/public/HappoDiffs.jsx

```diff
@@ -114,6 +115,69 @@ NewImages.propTypes = {
   images: PropTypes.arrayOf(PropTypes.shape(imageShape)).isRequired,
 };
 
+class Swiper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cursorLeftPercent: 50, // Start in the center
+    };
+    this.handleMouseMove = this.handleMouseMove.bind(this);
+  }
+
+  handleMouseMove(event) {
+    const leftPx = event.pageX - event.target.offsetLeft;
+    const leftPercent = leftPx / event.target.offsetWidth * 100;
+
+    this.setState({
+      cursorLeftPercent: leftPercent,
+    })
+  }
+
+  render() {
+    const { previous, current } = this.props;
+    const { cursorLeftPercent } = this.state;
+
+    const polygonPoints = [
+      `${cursorLeftPercent}% 0`, // top leftmost point
+      '100% 0', // top right corner
+      '100% 100%', // bottom right cornder
+      `${cursorLeftPercent}% 100%`, // bottom leftmost point
+    ];
+    const clipPath = `polygon(${polygonPoints.join(',')})`;
+
+    return (
+      <div
+        className='Swiper'
+        style={{
+          backgroundImage: `url(${previous})`,
+        }}
+        onMouseMove={this.handleMouseMove}
```

Comment:
What do you think about making this so it only moves when the mouse button is down?